keyboards/keychron/q0/rev_0130/keymaps/via/config.h

```diff
@@ -14,25 +14,4 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#pragma once
-
-#include "quantum.h"
-
-#define XXX KC_NO
-
-#define LAYOUT_numpad_6x4( \
-       K00, K01, K02, K03, \
-       K10, K11, K12, K13, \
-       K20, K21, K22,      \
-       K30, K31, K32, K23, \
-       K40, K41, K42,      \
-       K50,      K52, K43  \
-) \
-{ \
-    { K00, K01, K02, K03 }, \
-    { K10, K11, K12, K13 }, \
-    { K20, K21, K22, K23 }, \
-    { K30, K31, K32, XXX }, \
-    { K40, K41, K42, K43 }, \
-    { K50, XXX, K52, XXX }, \
-}
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
```

Comment:
This is the default value, and shouldn't need to be specified.

Reply:
All right, deleted it, thanks~